mne/tests/test_coreg.py

```diff
@@ -279,3 +279,27 @@ def test_get_mni_fiducials():
     fids_est = np.array([f['r'] for f in fids_est])
     dists = np.linalg.norm(fids - fids_est, axis=-1) * 1000.  # -> mm
     assert (dists < 8).all(), dists
+
+
+@testing.requires_testing_data
+def test_coregistration():
+    """Test automated coregistration."""
```

Comment:
can you test with the different scaling options?

Reply:
This is still pending.

For scalings you could for example test first that even with scalings enabled, you still end up with a similar head<->MRI trans and near-unity scale factors.

Then you can test that if you for example:
```
for d in info['dig']:
    d['r'] *= [0.8, 1.1, 1.1]
```
you get approximately these scale factors back from the automated routine. (Or maybe their inverse, not sure). I chose Y=Z here because IIRC the x axis is pretty well aligned between the MRI and head, but there is a rotation about the X axis, so the Y and Z scale factors will end up mixed, so setting them the same way just makes things easier.